From 50562d12dd747e149cf05fa9471162542fd9e589 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Tue, 27 Jan 2015 10:25:25 +0800
Subject: [PATCH 0086/5242] MLK-10132-4 usb: chipidea: imx: disable usb wakeup
 in probe

commit  62a0c1d0582ebb57ee32c7f5b761a99ea9cf233d from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable usb wakeup as initial setting in probe.

Signed-off-by: Li Jun <jun.li@freescale.com>
(cherry picked from commit bf54f23766bcebd781f1c09bd68bc00d790160c8)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/chipidea/ci_hdrc_imx.c |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/usb/chipidea/ci_hdrc_imx.c b/drivers/usb/chipidea/ci_hdrc_imx.c
index 58ba67b..453c46b 100644
--- a/drivers/usb/chipidea/ci_hdrc_imx.c
+++ b/drivers/usb/chipidea/ci_hdrc_imx.c
@@ -599,6 +599,12 @@ static int ci_hdrc_imx_probe(struct platform_device *pdev)
 		goto disable_device;
 	}
 
+	ret = imx_usbmisc_set_wakeup(data->usbmisc_data, false);
+	if (ret) {
+		dev_err(&pdev->dev, "usbmisc set_wakeup failed, ret=%d\n", ret);
+		goto disable_device;
+	}
+
 	/* usbmisc needs to know dr mode to choose wakeup setting */
 	data->usbmisc_data->available_role =
 			ci_hdrc_query_available_role(data->ci_pdev);
-- 
1.7.9.5

